src/AndersonAcceleration.f90

```diff
@@ -85,7 +90,7 @@ SUBROUTINE init_AndersonAccelerationType(solver,ce,Params)
   CHARACTER(LEN=*),PARAMETER :: myName='init_AndersonAccelerationType'
   CLASS(AndersonAccelerationType),INTENT(INOUT) :: solver
   TYPE(ParamType),INTENT(IN) :: Params
-  TYPE(FutilityComputingEnvironment),INTENT(IN) :: ce
+  TYPE(FutilityComputingEnvironment),TARGET :: ce
```

Comment:
Don't we still need intent on this?

Reply:
Let's put it in. We are missing Intent on the futility computing env in multiple locations in MPACT (was just emulating an init routine in MPACT).